.github/workflows/compile-examples.yml

```
@@ -51,6 +51,14 @@ jobs:
               - name: arduino:samd
             libraries: |
               - name: WiFiNINA
+          - fqbn: arduino:mbed:envie_m7
+            platforms: |
+              - name: arduino:mbed
```

```suggestion
          - fqbn: arduino:mbed_portenta:envie_m7
            platforms: |
              - name: arduino:mbed_portenta
```

The `arduino:mbed` platform is deprecated. So, unless there is a specific reason to test against that platform, it is better to use the preferred `arduino:mbed_portenta` platform.

---

No specific reason, just a mistake. Thanks @per1234